spec.emu

```diff
@@ -174,6 +174,9 @@ location: https://tc39.es/proposal-logical-assignment
         1. Perform ? DestructuringAssignmentEvaluation of _assignmentPattern_ using _rval_ as the argument.
         1. Return _rval_.
       </emu-alg>
+      <emu-note>
+        <p>When an assignment occurs within strict mode code, it is a runtime error if _lref_ in step 1.e is an unresolvable reference. If it is, a *ReferenceError* exception is thrown. The |LeftHandSideExpression| also may not be a reference to a data property with the attribute value { [[Writable]]: *false* }, to an accessor property with the attribute value { [[Set]]: *undefined* }, nor to a non-existent property of an object for which the IsExtensible predicate returns the value *false*. In these cases a *TypeError* exception is thrown.</p>
```

Comment:
I would say "When _this expression_ occurs..." or "When an assignment _of this form_ occurs..." or something like that. The "when an assignment occurs" wording made sense when the one note covered all assignments, but now there's different notes for different kinds of assignments.

Otherwise LGTM.